packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -262,6 +268,7 @@ export class SummarizerNode implements IRootSummarizerNode {
             correlatedSummaryLogger,
             readAndParseBlob,
         );
+        return false;
```

Comment:
Do we wanna short-circuit for oldest snapshots? sequence number comparison?

Reply:
That way `pendingSummary` can be set to true or something that bypasses downloading the snapshot..

Reply:
Doesn't that become a third case then? Basically, we did not have a pending summary but we didn't update the latest summary either.

Reply:
Maybe a better way would be for `refreshSummaryAck` in summarizer node to return whether this ack was used to update latest summary or not. If it was, then we download the snapshot and update `summaryGCVersion` based on the metadata blob. If it was not used, we simply ignore it.
Although its a little expensive because in the common case, for each ack, we will download the summary.

Reply:
@arinwt Updated this as per our conversation. Can you please take a look.